1-js/01-getting-started/1-intro/article.md

```diff
@@ -85,7 +85,7 @@ There are at least *three* great things about JavaScript:
 
 ```compare
 + Full integration with HTML/CSS.
-+ Simple things done simply.
```

Comment:
I think it's technically _correct_, but it feels off when to compared to the other points on this list.